whisper_ner/dataset.py

```diff
@@ -0,0 +1,307 @@
+import json
+import logging
+import random
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, List, Union
+
+import torch
+import torchaudio
+from transformers import WhisperProcessor
+
+
+from whisper_ner.utils import set_logger, token_padding
+
+set_logger()
```

Comment:
```suggestion
from whisper_ner.utils import token_padding
```

The logger will be set from `trainer.py` no?